pkg/safeguards/safeguards_constants.go

```diff
@@ -27,6 +27,16 @@ type Safeguard struct {
 	constraintPath string
 }
 
+type ManifestFile struct {
```

Comment:
are these constants? they seem like types so the file name could be misleading

Reply:
you're right